Description: Change the makefile to build the _semanage.so files
 The pkg-config program doesn't work so I hacked it out and hard coded the
 necessary python rules to do the build.
 .
Author: Russell Coker <russell@coker.com.au>

---

Last-Update: 2012-02-28

--- libsemanage-2.1.6.orig/src/Makefile
+++ libsemanage-2.1.6/src/Makefile
@@ -10,8 +10,8 @@ LIBDIR ?= $(PREFIX)/lib
 SHLIBDIR ?= $(PREFIX)/lib
 INCLUDEDIR ?= $(PREFIX)/include
 PYLIBVER ?= $(shell $(PYTHON) -c 'import sys;print("python%d.%d" % sys.version_info[0:2])')
-PYINC ?= $(shell pkg-config --cflags $(PYPREFIX))
-PYTHONLIBDIR ?= $(shell pkg-config --libs $(PYPREFIX))
+PYINC ?= -I/usr/include/$(PYPREFIX)
+PYTHONLIBDIR ?= /usr/lib/lib$(PYPREFIX).so
 PYLIBDIR ?= $(LIBDIR)/$(PYLIBVER)
 RUBYLIBVER ?= $(shell ruby -e 'print RUBY_VERSION.split(".")[0..1].join(".")')
 RUBYPLATFORM ?= $(shell ruby -e 'print RUBY_PLATFORM')
